crates/rethnet_eth/src/remote/methods/hardhat/compiler_io.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(missing_docs)]
```

Comment:
Why don't you add docs? New code should be written with docs. I've only included this for old code as we're slowly moving towards all code having docs

Reply:
these types were public back when they were in `rethnet_eth`, but now that i've moved them to `rethnet_rpc_server` (in https://github.com/NomicFoundation/hardhat/pull/3998/commits/f2fd69a0b195853f6019abac23e8ad3e6e22152c), i removed this directive in https://github.com/NomicFoundation/hardhat/commit/c86e0d6425c249174b9ddfa0630dff480663b03b